cluster/manifests/postgres-operator/config.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: postgres-operator
+  namespace: kube-system
+  labels:
+    application: postgresql-operator
+data:
+  service_account_name: operator
+  cluster_labels: application:spilo
+  cluster_name_label: version
+  pod_role_label: spilo-role
+  db_hosted_zone: teapot.zalan.do
+  debug_logging: "true"
+  dns_name_format: '%s.%s.staging.%s'
+  docker_image: registry.opensource.zalan.do/acid/spiloprivate-9.6:1.2-p7
+  etcd_host: etcd-client.default.svc.cluster.local:2379
```

Comment:
this looks wrong: the postgres-operator runs in kube-system, but etcd in default namespace?